vars/mailer.groovy

```diff
@@ -6,34 +6,48 @@ def sendEmailFailure() {
             recipientProviders: [[$class: 'DevelopersRecipientProvider'], [$class: 'RequesterRecipientProvider']])
 }
 
-def sendEmail_failedPR() {
+def sendEmail_failedPR(pr_type) {
```

Comment:
The problem here is other projects are using the method already
https://github.com/kiegroup/kogito-runtimes/blob/f4c84914b945e56c34d20f3ea88c725d7bfab968/Jenkinsfile#L75
So I would add pr_type as an optional argument plus the null check `${pr_type ?: ''}`